aip/cloud/index.md

```diff
@@ -7,9 +7,9 @@ js:
 permalink: /cloud
 ---
 
-# Cloud AIPs
+# Google Cloud Platform AIPs
 
-The following AIPs apply to work on APIs in the Google Cloud PA.
```

Comment:
Here too, I would prefer a separate `cli` directory and index.
One big reason for this is `CODEOWNERS` -- the gcloud team should be approvers on their block, but not the 2500 block.